src/formatDate.js

```diff
@@ -50,7 +50,63 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+  const splittedDate = date.split(oldSeparator);
+  let year = 0;
+  let month = 0;
+  let day = 0;
+  const formattedDate = [];
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat[i] === 'YYYY') {
+      year = splittedDate[i];
+    }
+
+    if (fromFormat[i] === 'YY') {
+      year = splittedDate[i];
+    }
```

Comment:
You can combine these conditions in one and also do the transformations from YYYY to YY or vice versa here

Reply:
tried my best 💪